init_app/init_data.go

```diff
@@ -0,0 +1,69 @@
+package init_app
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/google/go-github/v50/github"
+	"github.com/ministryofjustice/cloud-platform-go-get-module/utils"
+)
+
+var owner = "ministryofjustice"
+
+func InitDataClient(dataAddr, dataPassword string) utils.DataAccessLayer {
+	return initRedis(dataAddr, dataPassword)
+}
+
+func InitData(dataClient utils.DataAccessLayer) {
+	client := github.NewClient(nil)
+	repos, err := getRepos(client)
+
+	if err != nil {
+		log.Println("Error getting repo names: ", err)
```

Comment:
Can we return the error from this function and then print the output from the caller?

Reply:
Good point I've pushed a fixup.

Reply:
https://github.com/ministryofjustice/cloud-platform-go-get-module/pull/2/commits/dd410ed43e9518a9afd586dd7ea4b3427b3d9206